subsys/bluetooth/controller/ll_sw/lll_filter.h

```diff
@@ -10,21 +10,31 @@
 #define FAL_SIZE 8
 #endif /* CONFIG_BT_CTLR_FAL_SIZE */
```

Comment:
Remove definition of `FAL_SIZE` from code.

Reply:
done